ssh/config.json

```diff
@@ -1,10 +1,11 @@
 {
-  "name": "SSH server",
-  "version": "8.3",
+  "name": "Terminal & SSH",
+  "version": "8.4",
   "slug": "ssh",
   "description": "Allow logging in remotely to Home Assistant using SSH",
```

Comment:
This description is no longer accurate.  By default the add-on no longer **allows logging in remotely using ssh**

Reply:
Feel free to open up a PR to improve a thing, or open up an issue. Let's not use merged PR's for that, as it is not trackable. Thanks 👍